Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs

```diff
@@ -40,25 +58,32 @@ public static class WebApiServiceCollectionExtensions
             string configSectionName = "AzureAd",
             bool subscribeToJwtBearerMiddlewareDiagnosticsEvents = false)
         {
-            services.AddAuthentication(AzureADDefaults.JwtBearerAuthenticationScheme)
-                    .AddAzureADBearer(options => configuration.Bind(configSectionName, options));
-            services.Configure<AzureADOptions>(options => configuration.Bind(configSectionName, options));
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+                    .AddJwtBearer((options => configuration.Bind(configSectionName, options)));
+                  
+            services.Configure<MicrosoftIdentityOptions>(options => configuration.Bind(configSectionName, options));
 
             services.AddHttpContextAccessor();
 
             // Change the authentication configuration to accommodate the Microsoft identity platform endpoint (v2.0).
-            services.Configure<JwtBearerOptions>(AzureADDefaults.JwtBearerAuthenticationScheme, options =>
+            services.Configure<JwtBearerOptions>(JwtBearerDefaults.AuthenticationScheme, options =>
             {
+                var microsoftIdentityOptions = configuration.GetSection(configSectionName).Get<MicrosoftIdentityOptions>();
+
                 // Reinitialize the options as this has changed to JwtBearerOptions to pick configuration values for attributes unique to JwtBearerOptions
                 configuration.Bind(configSectionName, options);
```

Comment:
>configuration.Bind(configSectionName, options); [](start = 16, length = 47)

set a web api settings in appsettings and check if we still need this line of code. We no longer of two separate set of option classes

Reply:
Indeed this was not required anymore. Removed